governing-documents/project-annual-review.md

```diff
@@ -0,0 +1,92 @@
+---
+layout: default
+title: Project Annual Review
+parent: Governing Documents
+grand_parent: Hyperledger TOC
+nav_order: 5
+---
+[//]: # (SPDX-License-Identifier: CC-BY-4.0)
+
+# Project Annual Review
+
+## Overview
+In addition to the [quarterly project updates](./project-updates.md), projects will undergo an annual review by the TOC. Unlike the quarterly updates, the annual review process will include a deep dive into the project health, including contributor and maintainer diversity and adoption metrics. The annual review will also include an assessment of the current [project lifecycle state](./project-lifecycle.md). The review will result in a set of recommendations for the project to improve and/or recommendation to move a project to another stage.
+
+## Timeline
+Reviews will start on the yearly anniversary of the project being accepted or moving to a new stage.  **If your annual review is not submitted within two months of notification, we will take this as a sign that the project is not under active maintenance and the TOC is likely to decide to archive the project and move it to End of Life status.**
+
+**NOTE:** If a project has genuinely stalled we can save everyone’s time and effort by archiving it.
```

Comment:
Archiving implies End-of-Life, right? (Just trying to map this to the project lifecycle.)